src/lib/hooks/useMinimalShellMode.tsx

```diff
@@ -12,22 +13,27 @@ export function useMinimalShellMode() {
   }
 
   React.useEffect(() => {
-    if (store.shell.minimalShellMode) {
-      Animated.timing(minimalShellInterp, {
-        toValue: 1,
-        duration: 150,
-        useNativeDriver: true,
-        isInteraction: false,
-      }).start()
-    } else {
-      Animated.timing(minimalShellInterp, {
-        toValue: 0,
-        duration: 150,
-        useNativeDriver: true,
-        isInteraction: false,
-      }).start()
-    }
-  }, [minimalShellInterp, store.shell.minimalShellMode])
```

Comment:
The fix is that `store.shell.minimalShellMode` isn't read on render anymore, so MobX won't subscribe this component to its changes. We don't need to subscribe the component to its changes because we only care about re-running the imperative code, which `reaction` will do for us.